<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyStudentWidget</class>
 <widget class="QWidget" name="ModifyStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>69</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mathLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数学分析：</string>
   </property>
  </widget>
  <widget class="QLabel" name="enLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>英语四级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLabel">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C++：</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入学号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>209</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mathLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
